test/CMakeLists.txt

```diff
@@ -62,6 +63,11 @@ macro(add_gtest TESTNAME)
       target_compile_definitions(${TESTNAME} PRIVATE FBGEMM_STATIC)
     endif()
   else(MSVC)
+    if(MINGW)
+    #if (FBGEMM_LIBRARY_TYPE STREQUAL "static")
+      target_compile_definitions(${TESTNAME} PRIVATE FBGEMM_STATIC)
```

Comment:
I assume the macro should not be defined for all cases. That static library condition should be present, right?